src/ArborX_LinearBVH.hpp

```diff
@@ -57,18 +63,24 @@ class BoundingVolumeHierarchy
   KOKKOS_FUNCTION
   bounding_volume_type bounds() const noexcept { return _bounds; }
 
-  template <typename ExecutionSpace, typename Predicates, typename... Args>
+  template <typename ExecutionSpace, typename Predicates, typename Callback>
   void query(ExecutionSpace const &space, Predicates const &predicates,
-             Args &&... args) const
+             Callback const &callback,
+             Experimental::TraversalPolicy const &policy =
+                 Experimental::TraversalPolicy()) const;
+
+  template <typename ExecutionSpace, typename Predicates,
+            typename CallbackOrView, typename View, typename... Args>
+  // clang-format off
+  [[deprecated( "For crs format output, use a free function ArborX::query instead.")]]
```

Comment:
```suggestion
  [[deprecated( "For CRS format output, use the ArborX::query free function instead.")]]
```

Reply:
The deprecation was removed as suggested below.